collectors/feed/models.py

```diff
@@ -0,0 +1,28 @@
+from typing import NamedTuple
+
+
+class FeedEntry(NamedTuple):
+    """
+    Temporary container for feed entries before they are saved as new Snippet model instances.
+    """
+
+    url: str
+    cves: list
+    title: str
+    text: str
+    name: str
+    source: str
+    components: list
```

Comment:
Do we use that for anything?

Reply:
Aha. So I'll use the static per-feed list of components and give each snippet of that feed the same list of components, leaving it up to the analyst to decide whether all of them are relevant? This is the relevant part of the per-feed configuration: https://github.com/RedHatProductSecurity/osidb/blob/8b01f4604fbb6f2270e6fa4b5b75d49ba012333e/collectors/feed/config.yml#L45-L47

Reply:
And 2nd example: https://github.com/RedHatProductSecurity/osidb/blob/8b01f4604fbb6f2270e6fa4b5b75d49ba012333e/collectors/feed/config.yml#L82-L84

Reply:
Done